configuration/concepts/lab_tests.csv

```diff
@@ -61,7 +67,6 @@ c288ec60-4755-4f42-a4de-e197bf9ba0fa,,,Antistreptolysin O,Antiestreptolisina O,A
 00df2515-b7c2-45bb-ae5e-8872e9f2446f,,,Amylase,Amilasa,Amilasa,Amylase,,LabTest,Numeric,,,,,,,,,,
 6b12c39e-dfa8-4cdf-aa05-ca7e577d2fd2,,,Lipase,Lipasa,Lipasa,Lipase,,LabTest,Numeric,,,,,,,,,,
 3c42b57e-2b96-48d1-8f54-eb325d268882,,,Rheumatoid Factor,Factor reumatoide,Factor reumatoide,Rheumatoid Factor,,LabTest,Coded,,Positive;Negative,,,,,,,,
-c288ec60-4755-4f42-a4de-e197bf9ba0fa,,,Antistreptolysin O,Antiestreptolisina O,Antiestreptolisina O,Antistreptolysin O,,LabTest,Coded,,Positive;Negative,,,,,,,,

```

Comment:
Why did you delete this?

Reply:
line 60 was duplicated at line 64 which was deleted